pallets/did/src/benchmarking.rs

```diff
@@ -16,6 +16,8 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
+use core::ops::Mul;
```

Comment:
`sp_std`

Reply:
Updated in: [d0116a1](https://github.com/KILTprotocol/kilt-node/pull/655/commits/d0116a1bac911dafea111d3ae1e7af1a208f68e7)